docs/monitors/collectd-systemd.md

```diff
@@ -0,0 +1,96 @@
+    - ubuntu-fan
+  extraDimensions:
+    metric_source: collectd/systemd
+```
+
+
+Monitor Type: `collectd/systemd`
+
+[Monitor Source Code](https://github.com/signalfx/signalfx-agent/tree/master/internal/monitors/collectd/systemd)
+
+**Accepts Endpoints**: **Yes**
+
+**Multiple Instances Allowed**: Yes
+
+## Configuration
+
+| Config option | Required | Type | Description |
+| --- | --- | --- | --- |
+| `services` | **yes** | `list of strings` | Systemd services to report on |
+| `serviceStates` | no | `map of bools` | Systemd service states. The default state is `ActiveState` and the default metric exported is `gauge.substate.running`. Possible service states are `ActiveState`, `SubState` and `LoadState` |
+
+
+
+
+## Metrics
+
+The following table lists the metrics available for this monitor. Metrics that are marked as Included are standard metrics and are monitored by default.
+
+| Name | Type | Included | Description |
+| ---  | ---  | ---    | ---         |
+| `gauge.active_state.active` | gauge |  | Indicates that the systemd unit/service is active |
+| `gauge.active_state.inactive` | gauge |  | Indicates that the systemd unit/service is inactive and the previous run was successful or no previous run has taken place yet |
+| `gauge.active_state.activating` | gauge |  | Indicates that the systemd unit/service has previously been inactive but is currently in the process of entering an active state |
+| `gauge.active_state.deactivating` | gauge |  | Indicates that the systemd unit/service is currently in the process of deactivation |
+| `gauge.active_state.reloading` | gauge |  | Indicates that the systemd unit/service is active and currently reloading its configuration |
+| `gauge.active_state.failed` | gauge |  | Indicates that the systemd unit/service is inactive the previous run was not successful |
+| `gauge.substate.running` | gauge | ✔ | Indicates that the systemd unit/service is running |
+| `gauge.substate.exited` | gauge |  | Indicates that the systemd unit/service exited |
+| `gauge.substate.failed` | gauge |  | Indicates that the systemd unit/service failed |
+| `gauge.substate.dead` | gauge |  | Indicates that the systemd unit/service died |
+| `gauge.load_state.loaded` | gauge |  | Indicates that the systemd unit/service configuration was loaded and parsed successfully |
+| `gauge.load_state.not-found` | gauge |  | Indicates that the systemd unit/service configuration was not found |
+| `gauge.load_state.error` | gauge |  | Indicates that the systemd unit/service configuration failed to load |
+| `gauge.load_state.masked` | gauge |  | Indicates that the systemd unit/service is currently masked out (i.e. symlinked to /dev/null etc) |
+
+
+
+## Dimensions
+
+The following dimensions may occur on metrics emitted by this monitor.  Some
+dimensions may be specific to certain metrics.
+
+| Name | Description |
+| ---  | ---         |
+| `plugin_instance` | The name of the systemd service that the reported metric applies to |
```

Comment:
Can we not just call it `systemd_service` or something a million times more descriptive than `plugin_instance`?